README.md

```diff
@@ -2,31 +2,37 @@
 
 # crystal-db
 
-Common db api for crystal. You will need to a specific driver to access a database.
+Common DB API for Crystal. You will need a specific driver to access a database.
 
-* [sqlite](https://github.com/crystal-lang/crystal-sqlite3)
-* [mysql](https://github.com/crystal-lang/crystal-mysql)
-* [postgres](https://github.com/will/crystal-pg)
+* [SQLite](https://github.com/crystal-lang/crystal-sqlite3)
+* [MySQL](https://github.com/crystal-lang/crystal-mysql)
+* [PostgreSQL](https://github.com/will/crystal-pg)
 
 ## Installation
 
-Add this to your application's `shard.yml`:
+Add `crystal-db` to your application's `shard.yml`:
 
 ```yaml
 dependencies:
   db:
     github: crystal-lang/crystal-db
 ```
 
+Add the driver for your database, i. e. for Postgres:
+
+```yaml
+dependencies:
+  pg:
+    github: will/crystal-pg
```

Comment:
given that the usage sample is with sqlite (which requires less setup), I would suggest using sqlite here also. And I think is more clear to have only one yaml section with all the content. Otherwise it seems like the `db:` shard was removed here.

`i. e.` -> `i.e.` or `ie.`